go/filer/flat_namespace/flat_namespace_store.go

```diff
@@ -1,9 +1,16 @@
 package flat_namespace
 
-import ()
+import "github.com/chrislusf/seaweedfs/go/filer"
 
 type FlatNamespaceStore interface {
 	Put(fullFileName string, fid string) (err error)
 	Get(fullFileName string) (fid string, err error)
 	Delete(fullFileName string) (fid string, err error)
+
+	//redis store support these functions
```

Comment:
FlagNamespaceStore means just fullpath-to-fid mapping.

Please create a new interface for these operations.